docs/examples/python-requests-jwt.md

```diff
@@ -0,0 +1,49 @@
+## Accessing PostgREST API using Python Requests and Requests-JWT
+
+This code relies on setting up the PostgreSQL auth functions and grants correctly first. Follow these instructions:
+
+http://postgrest.com/examples/users/
+
+After completing the configuration, be sure to create a user with email, password, role, and verified flag. We'll use that user to login in the code below.
+
+Install the required libraries
+
+    pip install requests
+    pip install requests-jwt
+
+Then, from a python interpreter or script:
+
+    import requests
+    import requests_jwt
+    import json
+
+    # email and password auth through postgrest
+    resp = requests.post('http://localhost:3000/rpc/login', json={"email": "you@yours.com", "pass": "dog"})
+    if resp.status_code != 200:
+        raise Exception()
+
+    # JWT auth using the token above
+    token = json.loads(resp.text)['token']
+    auth = requests_jwt.JWTAuth(token)
+    if r.status_code != 200:
+        raise Exception()
+
+    # Make a request to get the range header
+    r = requests.get('http://localhost:3000/posts', auth=auth)
```

Comment:
The spec is pretty confusing in this regard. I've read various versions of the spec and I can't quite find the clarity that I'm seeking. Turns out that the HATEOAS folks and RFC 5988 have produced some thoughtful inquiry on the subject but nothing I'd describe as authoritative.

http://pilhuhn.blogspot.com/2013/02/best-practice-for-paging-in-restful-apis.html

The notion of next and previous links seems usable. I believe that the key issue for clients is: what is the most intuitive way to interact with the API and make decisions about how to proceed through a paginated set? I'm scratching my head wondering how we've gotten this far without a standard for what seem like such rudimentary operations.